src/v2i-hub/TelematicBridgePlugin/src/TelematicBridgePlugin.cpp

```diff
@@ -0,0 +1,39 @@
+#include "TelematicBridgePlugin.h"
+
+namespace TelematicBridge
+{
+    TelematicBridgePlugin::TelematicBridgePlugin(const string &name) : PluginClient(name)
+    {
+        AddMessageFilter("*", "*", IvpMsgFlags_None);
```

Comment:
Will this only pickup messages with no IvpMsgFlags? I imagine this filter should look essentially identical to the ImmediateForward filter, except it should not require a DSRC flag. 

The immediate forward filer looks like this :
```cpp
AddMessageFilter("J2735", "*", IvpMsgFlags_RouteDSRC);
```

Reply:
How do we map the PSID to each TMX message? eg: time sync TMX message, do we have a PSID for that? Which messages use the IvpMsgFlags_RouteDSRC flag? When I test the J2735 message with Message receiver plugin, I can subcribe to the J2735 without IvpMsgFlags_RouteDSRC flag. The intention for this plugin is to discover all messages. So that the plugin can tell telematic system what topics/message available in V2xHub. V2xHub UI does not control what to subscribe to. Instead, the telematic UI will receive a list of all available topics/messages  and decide which messages to send to cloud. Note: we will create a mapping between messages and topics as  the plugin discover messages with the AddMessageFilter func.

Reply:
Immediate Forward plugin subscribe to all J2735 message and use IVPMessage rather than routable message.